README.md

```diff
@@ -70,7 +74,8 @@ Sending requests
       --data '{"descriptor":"some-channel","token":"oqiwdJKNsdKIUwezd92DNQsndkDERDFKJNQWSwq3rODIU",
               "sender":{"email":"your-name@company.com","name":"Sender"},
               "recipients":[{"email":"your-client@client.com","name":"Recipient"}],
-              "cc": [],"bcc": [],"min_period":0.0001,"max_size":10000000}' "localhost:8300/api/channel"
+              "cc": [],"bcc": [],"min_period":0.0001,"max_size":10000000}' 
```

Comment:
I think you need a backslash here? (and also on the lines before which are not part of the single-quote block ( `'...'`)